### ISSUE TITLE
Cross Platform?

### ISSUE THREAD

**Issue by markrickert** [GantMan/yesyoucam#2]
Thoughts on making this cross-platform so that we can not have to use the bubblewrap camera library? :P

**Comment by GantMan** [comment]
Yes!  I was just talking to someone about that.   I'm hoping to find a need sometime soon and then kick it's ass!
